tez/tez.sh

```diff
@@ -132,14 +135,24 @@ function configure_master_node() {
   systemctl restart hadoop-yarn-timelineserver
   systemctl status hadoop-yarn-timelineserver  # Ensure it started successfully
 
-  # Restart hive server2
-  systemctl restart hive-server2
-  systemctl status hive-server2  # Ensure it started successfully
+  # Check hive-server2 status
+  local hive_server2_status
+  hive_server2_status=$(systemctl show --property=LoadState hive-server2.service \
```

Comment:
This will work for hive-server2 because it's either installed or not installed. I'm worried about creating a precedent that won't work for namenodes -- on `-m-2`, the namenode package is installed but not enabled.

I'd prefer to use `systemctl list-unit-files --state=enabled | grep <whatever>.service` to only worry about services we plan to run.

Reply:
Yes, I checked that and your proposal will work only for native services, so for instance `hive-server2.sevice` will not appear on the list when checking using `systemctl list-unit-files --state=enabled` command. It will be on the list when we search using `systemctl list-unit-files --type=service` however by its state in the output we can't determine its right status. 
I think that, if we could consider that services we plan to run must be enabled (to allow them automatically start after the reboot ), then the nicest way to check whether service is enabled is by running: 
`systemctl is-enabled --quiet hive-server2.service`

Reply:
Awesome, didn't know about that one!